### PR TITLE
Document Lambda layers for GovCloud

### PR DESCRIPTION
**Pull request by tianchu**
### What does this PR do?
Document Lambda layers for GovCloud

### Motivation
Allow GovCloud customers to use the prebuilt Lambda layers

### Preview link

https://docs-staging.datadoghq.com/tian.chu/lambda-layer-for-govcloud/serverless/installation/python/?tab=custom
https://docs-staging.datadoghq.com/tian.chu/lambda-layer-for-govcloud/serverless/installation/nodejs/?tab=custom

### CODE REVIEW

File: content/en/serverless/_index.md [hunk 1]
```diff
@@ -29,27 +29,36 @@ Serverless is a concept where you write event-driven code and upload it to a clo
...
 
-Start by installing the [AWS Integration][2]. This allows Datadog to ingest Amazon CloudWatch metrics from AWS Lambda. The AWS integration installation also configures the Datadog Forwarder, which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs.
+Install the [AWS integration][2]. This allows Datadog to ingest Lambda metrics from AWS CloudWatch. 
 
-### 2. Instrument your application
+### 2. Install the Datadog Forwarder
+
+Install the [Datadog Forwarder Lambda function][3], which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs.
+
+Note, if you installed the AWS integration by using the Datadog-provided CloudFormation template, then you already have the Forwarder function installed as part of the AWS integration CloudFormation stack.
```
Comment (Hesperide) [inline content/en/serverless/_index.md:38]: Can we modify the wording here so this step isn't followed by customers who installed the AWS integration? Maybe as part of section 1, we could have "If you did installed the AWS integration using the Datadog-provided CloudFormation template, install the [Datadog Forwarder Lambda function][3], which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs."

File: content/en/serverless/_index.md [hunk 2]
```diff
@@ -29,27 +29,34 @@ Serverless is a concept where you write event-driven code and upload it to a clo
 
 ### 1. Install the AWS Integration
```
Comment (ruthnaebeck) [inline content/en/serverless/_index.md:30]:
```suggestion
### 1. Install the AWS integration
```

File: content/en/serverless/_index.md [hunk 3]
```diff
@@ -29,27 +29,34 @@ Serverless is a concept where you write event-driven code and upload it to a clo
...
 
-Start by installing the [AWS Integration][2]. This allows Datadog to ingest Amazon CloudWatch metrics from AWS Lambda. The AWS integration installation also configures the Datadog Forwarder, which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs.
+Install the [AWS integration][2]. This allows Datadog to ingest Lambda metrics from AWS CloudWatch. 
 
-### 2. Instrument your application
+### 2. Install the Datadog Forwarder
```
Comment (ruthnaebeck) [inline content/en/serverless/_index.md:34]:
```suggestion
### 2. Install the Datadog forwarder
```

File: content/en/serverless/_index.md [hunk 4]
```diff
@@ -29,27 +29,34 @@ Serverless is a concept where you write event-driven code and upload it to a clo
...
+
+Skip this step if you already have the Forwarder function installed as part of the [AWS integration][2] CloudFormation stack. Otherwise, install the [Datadog Forwarder Lambda function][3], which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs.
```
Comment (ruthnaebeck) [inline content/en/serverless/_index.md:36]:
```suggestion
Install the [Datadog Forwarder Lambda function][3], which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs. **Note**: Skip this step if you already have the forwarder function installed as part of the [AWS integration][2] CloudFormation stack.
```

File: content/en/serverless/_index.md [hunk 5]
```diff
@@ -29,27 +29,34 @@ Serverless is a concept where you write event-driven code and upload it to a clo
...
+
+### 3. Instrument Your Application
```
Comment (ruthnaebeck) [inline content/en/serverless/_index.md:38]:
```suggestion
### 3. Instrument your application
```

File: content/en/serverless/_index.md [hunk 6]
```diff
@@ -29,27 +29,34 @@ Serverless is a concept where you write event-driven code and upload it to a clo
...
+
+Select the Lambda runtime below for instructions to instrument your serverless application.
 
 {{< partial name="serverless/getting-started-languages.html" >}}
 
 ## Not Using AWS Lambda?
```
Comment (ruthnaebeck) [inline content/en/serverless/_index.md:44]:
```suggestion
## Other services
```

File: content/en/serverless/_index.md [hunk 7]
```diff
@@ -29,27 +29,34 @@ Serverless is a concept where you write event-driven code and upload it to a clo
...
 
 ### Azure App Services
```
Comment (ruthnaebeck) [inline content/en/serverless/_index.md:46]:
```suggestion
### Azure App Service
```

File: content/en/serverless/_index.md [hunk 8]
```diff
@@ -29,27 +29,34 @@ Serverless is a concept where you write event-driven code and upload it to a clo
...
 
-The Datadog extension for Azure App Services provides tracing capabilities for Azure Web Apps. For more information setting up tracing in Azure, see the [Azure App Services Extension documentation][3].
+The Datadog extension for Azure App Services provides tracing capabilities for Azure Web Apps. For more information setting up tracing in Azure, see the [Azure App Services Extension documentation][4].
```
Comment (ruthnaebeck) [inline content/en/serverless/_index.md:48]:
```suggestion
The Datadog extension for Azure App Service provides tracing capabilities for Azure Web Apps. For more information setting up tracing in Azure, see the [Azure App Service Extension documentation][4].
```

File: content/en/serverless/installation/_index.md [hunk 9]
```diff
@@ -16,9 +16,7 @@ Install the [AWS integration][1]. This allows Datadog to ingest Lambda metrics f
 
 ### 2. Install the Datadog Forwarder
```
Comment (ruthnaebeck) [inline content/en/serverless/installation/_index.md:17]:
```suggestion
### 2. Install the Datadog forwarder
```

File: content/en/serverless/installation/_index.md [hunk 10]
```diff
@@ -16,9 +16,7 @@ Install the [AWS integration][1]. This allows Datadog to ingest Lambda metrics f
...
 
-Install the [Datadog Forwarder Lambda function][2], which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs.
-
-Note, if you installed the AWS integration by using the Datadog-provided CloudFormation template, then you already have the Forwarder function installed as part of the AWS integration CloudFormation stack.
+Skip this step if you already have the Forwarder function installed as part of the [AWS integration][1] CloudFormation stack. Otherwise, install the [Datadog Forwarder Lambda function][2], which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs.
```
Comment (ruthnaebeck) [inline content/en/serverless/installation/_index.md:19]:
```suggestion
Install the [Datadog Forwarder Lambda function][2], which is required for ingestion of AWS Lambda traces, enhanced metrics, custom metrics, and logs. **Note**: Skip this step if you already have the Forwarder function installed as part of the [AWS integration][1] CloudFormation stack.
```

File: content/en/serverless/_index.md (hunk 7 above)
Comment (tianchu) [reply content/en/serverless/_index.md:46]: "Azure App Services" is the official product name from Azure.

File: content/en/serverless/_index.md (hunk 3 above)
Comment (tianchu) [reply content/en/serverless/_index.md:34]: We do use `Forwarder` in all places wherever it's mentioned in the doc.

**Pull request merged**